metacat-client/src/main/java/com/netflix/metacat/client/module/MetacatErrorDecoder.java

```diff
@@ -77,7 +78,7 @@ public Exception decode(final String methodKey, final Response response) {
                     return new MetacatTooManyRequestsException(message);
                 case 500: //INTERNAL_SERVER_ERROR
                 case 503: //SERVICE_UNAVAILABLE
-                    return new RetryableException(message, null);
+                    return new RetryableException(message, null, new Date());
```

Comment:
Use the java.time.Instant class

Reply:
Ah this is a Feign API that requires a java.util.Date.

Reply:
I meant using  >> new Date(Instant.now().toEpochMilli()).

Reply:
Ah I see why, fixed, thanks.